libs/ui/elements/src/lib/thumbnail/thumbnail.component.ts

```diff
@@ -85,6 +88,7 @@ export class ThumbnailComponent implements OnInit, OnChanges {
   private setNewSrcImage(image: ThumbnailImageObject) {
     this.imgFit = image.fit
     this.imgUrl = image.url
+    this.placeholderShown.emit(this.isPlaceholder)
```

Comment:
Just wondering why this is not inside `setPlaceholder()` always returning `true`.

Reply:
I can answer for @Angi-Kinas since we've done this together: we wanted this output to emit either `true` or `false` according to the state of the thumbnail, and not just emit `true` when the placeholder shows up